pkgs/top-level/all-packages.nix

```diff
@@ -36046,10 +36046,9 @@ with pkgs;
   moon-buggy = callPackage ../games/moon-buggy { };
 
   minetest = callPackage ../games/minetest {
```

Comment:
Push an extra commit migrating this to by-name.

Reply:
Why? That’s by no means mandatory. No reason to block what is otherwise an improvement on that.

Reply:
I generally try avoid refactoring more than I need to when I make changes to code other people are working on. IMHO it also makes the review unnecessarily hard. In this case because moving the file to a new path can easily break the diff.

@AndersonTorres if you think this is a good idea,  we could do it in another PR after this one. After all, it should not cause any rebuilds if done correctly.

Reply:
> Why?

1. Less lines in all-packages.nix
2. One of the preconditions of the merge bot is packages in by-name
3. https://github.com/NixOS/nixpkgs/pull/334206#discussion_r1714470640

> That’s by no means mandatory. 

Not prohibited either (slightly encouraged in refactors).

> No reason to block what is otherwise an improvement on that.

Albeit I do not have true blocking power, I apologize -  it was not my intention to block this.